src/devtools/client/debugger/src/actions/event-listeners.js

```diff
@@ -103,8 +104,22 @@ export function getEventListenerBreakpointTypes() {
     dispatch({ type: "RECEIVE_EVENT_LISTENER_TYPES", categories });
 
     if (features.eventCount) {
-      const eventTypePoints = await client.fetchEventTypePoints(INITIAL_EVENT_BREAKPOINTS);
+      const { eventTypePoints, eventTypeResults } = await client.fetchEventTypePoints(
+        INITIAL_EVENT_BREAKPOINTS
+      );
+      console.log({ eventTypePoints, eventTypeResults });
       dispatch({ type: "RECEIVE_EVENT_LISTENER_POINTS", eventTypePoints });
```

Comment:
This is where we stopped before—we just added the `eventTypePoints` to Redux. That point data was then used in two places: 1) the hit count beside an event in the event dropdown, and 2) to match an event in the event panel (sidebar) to a point with a source.